generators/client/templates/angular/src/main/webapp/app/admin/admin.module.ts.ejs

```diff
@@ -54,7 +55,7 @@ import {
 @NgModule({
     imports: [
         <%=angularXAppName%>SharedModule,
-        RouterModule.forChild(adminState),
+        RouterModule.forChild(adminState)
```

Comment:
please retain the comma, otherwise there will be a compile error while adding new needles, else move the needle before this line